src/d_clisrv.c

```diff
@@ -2809,6 +2847,9 @@ void D_ClientServerInit(void)
 	COM_AddCommand("drop", Command_Drop);
 	COM_AddCommand("droprate", Command_Droprate);
 #endif
+#ifdef DEBUGMODE
```

Comment:
DEBUGMODE? did you mean _DEBUG? (same for the other two linked places with "DEBUGMODE")